api/src/main/java/uk/gov/hmcts/payment/api/configuration/LiberataOAuth2RestOperationsConfiguration.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.payment.api.configuration;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.oauth2.client.DefaultOAuth2ClientContext;
+import org.springframework.security.oauth2.client.OAuth2RestOperations;
+import org.springframework.security.oauth2.client.OAuth2RestTemplate;
+import org.springframework.security.oauth2.client.resource.OAuth2ProtectedResourceDetails;
+import org.springframework.security.oauth2.client.token.AccessTokenRequest;
+import org.springframework.security.oauth2.client.token.DefaultAccessTokenRequest;
+import org.springframework.security.oauth2.client.token.grant.password.ResourceOwnerPasswordResourceDetails;
+import org.springframework.security.oauth2.config.annotation.web.configuration.EnableOAuth2Client;
+
+@EnableOAuth2Client
+@Configuration
+public class LiberataOAuth2RestOperationsConfiguration {
+
+    @Value("${liberata.oauth2.token.url}")
```

Comment:
liberata have so many systems. we are dealing with pba system here. so ideally key names should be pba.token.url and pba.clientid and so on.